index.html

```diff
@@ -23,8 +23,8 @@ <h2>starUI🌟</h2>
   <main class='container-center'>
     <h2>Build fast, responsive sites with Bootstrap </h2>
     <p class="small-text">starUI is a free, open-source framework that provides ready-to-use frontend components that you can easily combine to build responsive interface. </p>
-    <button class='btn primary-btn small-text'><a>Get Started</a></button>
-    <button class='btn secondary-btn small-text'><a>GitHub</a></button>
+    <button class='btn primary-btn small-text'><a href="./pages/introduction/introduction.html">Get Started</a></button>
+    <button class='btn secondary-btn small-text'><a>GitHub</a></button>s
```

Comment:
**Fix**: Remove the "s" after the githug button.

Reply:
Thank you Vishal !!